db/migrate/20170116210702_add_manger_ref_to_authentications.rb

```diff
@@ -0,0 +1,5 @@
+class AddMangerRefToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :manager_ref, :string
```

Comment:
If name is currently the unique key for EMS Refresh, then when we have to do the following

1. I can merge this now, but...
2. Once the save_inventory code (and or graph refresh) is changed to use manager_ref over name, then all parsers that actually save data into this table will need to change.
3. There will also need to be a data migration for existing authentications from providers to map the name into the manager_ref.